packages/frontend/src/utils/twoFactor.js

```diff
@@ -48,6 +48,20 @@ export class TwoFactor extends Account2FA {
         });
     }
 
+    async getMultisigRequest() {
+        const request = this.getRequest();
+        const { receiver_id } = await this.viewFunction(
```

Comment:
ideally we could just use the result of `getRequest()` but it doesn't contain the `receiver_id` so we need to fetch this from the contract.